airflow/utils/cli.py

```diff
@@ -177,7 +177,7 @@ def get_dag_by_file_location(dag_id: str):
     dag_model = DagModel.get_current(dag_id)
     if dag_model is None:
         raise AirflowException(
-            f'dag_id could not be found: {dag_id}. Either the dag did not exist or it failed to parse.'
+            f"Dag '{dag_id}' could not be found; either it does not exist or it failed to parse."
```

Comment:
```suggestion
            f"Dag {dag_id!r} could not be found; either it does not exist or it failed to parse."
```

Reply:
It protects against [Log Injection](https://owasp.org/www-community/attacks/Log_Injection).